### PR TITLE
Removed ASE and changed varibles to move to New Shared ASP

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "appServicePlanName": {
+        "SharedFrontEndAppServicePlanName": {
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
             "type": "string"
         },
-        "appServicePlanResourceGroup": {
+        "SharedEnvResourceGroup": {
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 3]
```diff
@@ -154,10 +140,10 @@
                         "value": "[parameters('appServiceName')]"
                     },
                     "appServicePlanName": {
-                        "value": "[parameters('appServicePlanName')]"
+                        "value": "[parameters('SharedFrontEndAppServicePlanName')]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 4]
```diff
@@ -154,10 +140,10 @@
...
                     },
                     "appServicePlanResourceGroup": {
-                        "value": "[parameters('appServicePlanResourceGroup')]"
+                        "value": "[parameters('SharedEnvResourceGroup')]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 5]
```diff
@@ -194,7 +180,7 @@
                         "value": "[parameters('customHostName')]"
                     },
                     "certificateThumbprint": {
-                        "value": "[reference(resourceId(parameters('appServicePlanResourceGroup'), 'Microsoft.Web/certificates', parameters('keyVaultCertificateName')), '2016-03-01').Thumbprint]"
+                        "value": "[reference(resourceId(parameters('SharedEnvResourceGroup'), 'Microsoft.Web/certificates', parameters('keyVaultCertificateName')), '2016-03-01').Thumbprint]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 6]
```diff
@@ -215,10 +201,10 @@
                         "value": "[parameters('apiAppServiceName')]"
                     },
                     "appServicePlanName": {
-                        "value": "[parameters('appServicePlanName')]"
+                        "value": "[parameters('SharedFrontEndAppServicePlanName')]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 7]
```diff
@@ -215,10 +201,10 @@
...
                     },
                     "appServicePlanResourceGroup": {
-                        "value": "[parameters('appServicePlanResourceGroup')]"
+                        "value": "[parameters('SharedEnvResourceGroup')]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

File: azure/template.json [hunk 8]
```diff
@@ -251,7 +237,7 @@
                         "value": "[parameters('apiCustomHostName')]"
                     },
                     "certificateThumbprint": {
-                        "value": "[reference(resourceId(parameters('appServicePlanResourceGroup'), 'Microsoft.Web/certificates', parameters('apiKeyVaultCertificateName')), '2016-03-01').Thumbprint]"
+                        "value": "[reference(resourceId(parameters('SharedEnvResourceGroup'), 'Microsoft.Web/certificates', parameters('apiKeyVaultCertificateName')), '2016-03-01').Thumbprint]"
```
Comment (chelnak) [inline azure/template.json]: camelCase

**Comment by EwanNoble** [comment]
The API needs to be on the back end app service plan and  it will need certificate deployments adding

File: azure/template.json (hunk 1 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 2 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 4 above)
Comment (mwevill) [reply azure/template.json]: Fixed

File: azure/template.json (hunk 5 above)
Comment (mwevill) [reply azure/template.json]: fixed

**Pull request merged**